mantidproject/mantid
# Indirect Data Analysis - Undock mini plots

**Review on Windows required due to the dockwidget not being movable on linux**

**Description of work.**
This PR introduces the ability to undock the mini-plots seen on MSDFit, IqtFit, ConvFit and F(Q)Fit. This is a useful feature for the scientists using devices with small screens.

**To test:**
1. `Interfaces`->`Indirect`->`ConvFit` tab
2. Load the data below
3. Select a fit function (optional)
4. Click Run and wait (optional)
5. Double click the heading which says `Mini-plots` - the mini-plots should undock and can be resized.
6. Double click the heading again. The mini-plots should dock themselves back into their original position

The undocking feature should also work the same on MSDFit, IqtFit and F(Q)Fit.

**Data Files**
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2744513/irs26176_graphite002_red.zip)
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2744514/irs26173_graphite002_res.zip)

Fixes #24448 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
